torch/_prims/__init__.py

```diff
@@ -786,6 +788,13 @@ def _fill_aten(a: Tensor, value: NumberType) -> Tensor:
     type_promotion=ELEMENTWISE_PRIM_TYPE_PROMOTION_KIND.DEFAULT,
 )
 
+sgn = _make_elementwise_unary_prim(
```

Comment:
Hey @HaoZeke! What's your thinking on making both of these prims? Intuitively I'd suspect them to just be references